Resources/Prototypes/_Goobstation/Changeling/Entities/Guidebook/changeling.yml

```
@@ -0,0 +1,9 @@
+- type: entity
+  id: GuidebookChangelingFluff
+  name: guidebook changeling
+  description: you shouldn't be seeing this normally.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)